### PR TITLE
Allow newlines in unwrapped Labels

### PR DESCRIPTION
**Pull request by LeeARichardson**
See: https://github.com/picoe/Eto/issues/2326

### CODE REVIEW

**Comment by LeeARichardson** [comment]
This would be a breaking change so would you be interested if I do a separate PR to add an additional WrapMode for 2.x? `WrapMode.NoneMultiline` maybe?
I'm not able to test other platforms so is there any other work required?

File: test/Eto.Test/Sections/Layouts/TableLayoutSection/ScalingSection.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ public ScalingSection()
 			tableLayout.SetColumnScale(2);
 			tableLayout.SetRowScale(0);
 			tableLayout.SetRowScale(2);
-			tableLayout.Add(new Label { Text = "2x2, should not scale and be centered", BackgroundColor = Colors.Red }, 1, 1);
+			tableLayout.Add(new Label { Text = "3x3, should not scale and be centered", BackgroundColor = Colors.Red }, 1, 1);
```
Comment (cwensley) [inline test/Eto.Test/Sections/Layouts/TableLayoutSection/ScalingSection.cs:53]: ?  Why was this changed?

**Comment by cwensley** [comment]
Hey @LeeARichardson, thanks for submitting the PR!  Yes all platforms should behave the same, so if doing this brings Gtk and WinForms in line with Mac and WPF I'm all for it.

Adding a separate `WrapMode.Singleline` or something similar so one could set the label to be single line even though there are newlines might be useful, but it is equally as easy just to remove the newlines from the string.

**Comment by cwensley** [comment]
Just did a test and everything looks great.  

Thanks for the contribution! 😊
